### PR TITLE
Cynthia's About ME

### PR DESCRIPTION
**Pull request by candre717**
SI

### CODE REVIEW

**Comment by PatriciaMuench** [comment]
Looks pretty good! Maybe you could get the image to be smaller by giving it an id and then styling it in your css.  (But idk what size you should make it.)

File: cynthia.css [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  h2{
+    text-align: center
+  }
+
+  p{
+    text-align: center
+  }
+
+  #container {
+
```
Comment (chdastolfo) [inline cynthia.css]: I'm assuming this is a WIP and you're going to put style attributes in these empty ones - if not, you can delete them.

File: cynthia.css [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  body {
+    background-color: lightblue;
+
```
Comment (chdastolfo) [inline cynthia.css]: You can remove the extra space here.

**Comment by chdastolfo** [comment]
Nicely done! Just two things:

1. It looks like your css file is located in the main directory - you should put it in the CSS subfolder.
2. Once you've done that, you should add a link to cynthia.html to the nav bar on the index.html page.

**Pull request merged**